src/Color.php

```diff
@@ -280,4 +281,18 @@ private static function clamp($val, $min = 0, $max = 1)
     {
         return min($max, max($min, $val));
     }
+
+    /**
+     * HTML representation of color for usage in CMS	 * 
+     * @return string HTML code containing color and HEX color code
+     */
+    public function ColorCMS()
+    {
+
+        $colorhtml = '';
+        if($this->value) {
+            $colorhtml = '<span class="color-cms" style="display: inline-block; vertical-align: bottom; width: 20px; height: 20px; border-radius: 10px; background-color: #'.$this->value.'"></span> #'.$this->value;
```

Comment:
Since this is a varchar value (and thus can be arbitrary text) can you please use `Convert::raw2xml` to safely escape this property? I suggest to do this in both the attribute and the body text.

Reply:
I added the conversion for the color value to string in 5c4c67b